frontend/catalyst/compiler.py

```diff
@@ -248,13 +248,13 @@ def run_writing_command(command: List[str], compile_options: Optional[CompileOpt
         "convert-index-to-llvm",
         "convert-catalyst-to-llvm",
         "convert-quantum-to-llvm",
+        "add-exception-handling",
```

Comment:
Can you add a comment on why it is at that stage?

Reply:
Done! Thanks: https://github.com/PennyLaneAI/catalyst/pull/995/commits/a9a0d3cf8b17fffd256dad65ddc9ba8cce794d88